shell/components/nav/__tests__/Type.test.ts

```diff
@@ -1,167 +1,365 @@
-import { mount, RouterLinkStub } from '@vue/test-utils';
+import { shallowMount, RouterLinkStub, createLocalVue } from '@vue/test-utils';
 import Type from '@shell/components/nav/Type.vue';
+import { createChildRenderingRouterLinkStub } from '@shell/utils/unit-tests/ChildRenderingRouterLinkStub';
+import { TYPE_MODES } from '@shell/store/type-map';
+
+// Configuration
+const activeClass = 'router-link-active';
+const exactActiveClass = 'router-link-exact-active';
+const rootClass = 'root';
+const favoriteClass = 'favorite';
+const defaultCount = 1;
+const defaultRouteTypeProp = {
+  name:  'route-type',
+  route: 'route',
+  exact: true,
+  mode:  TYPE_MODES.FAVORITE
+};
+const defaultLinkTypeProp = {
+  link:   'link-type-link',
+  label:  'link-type-label',
+  target: 'link-type-target'
+};
 
 // Mandatory to mock vue-router in this test
 jest.mock('vue-router');
+const storeMock = {
+  getters: {
+    currentStore:    () => 'cluster',
+    'cluster/count': () => defaultCount,
+  }
+};
+const routerMock = { resolve: () => ({ route: { path: 'whatever' } }) };
+const localVue = createLocalVue();
```

Comment:
This is not needed.

Reply:
It looks like you're correct about routerMock. LocalVue is still needed per my other comment regarding cleanHtml.

Reply:
I will have to remove this on the migration anyway because [`createLocalVue` does not exist anymore](https://test-utils.vuejs.org/migration/#No-more-createLocalVue).
This should be rather mocked globally and tested in an isolated way.

Since I've seen that we have a dozen of tests like this, I think we can leave this.